FinniversKit/Sources/Base Components/ButtonStyle.swift

```diff
@@ -80,11 +80,13 @@ public struct CallToAction: ButtonStyle {
     @Binding var isEnabled: Bool
     private let background: Color
     private let font: Font
+    private let verticalPadding: CGFloat
     private let fullWidth: Bool
 
     public init(size: Button.Size = .normal, background: Color = .btnPrimary, fullWidth: Bool = true, isEnabled: Binding<Bool>? = nil) {
         self.background = background
         self.fullWidth = fullWidth
+        self.verticalPadding = size == .normal ? 13 : .spacingS
```

Comment:
Could you move 13 out to a constant or something? 🙂

Reply:
Not sure what I would call it, 13 is not an "official" kind of spacing, like spacingM, spacingS etc 🤔 
But could move it out to one called `normalButtonSpacing` or something like that?

Reply:
Hehe, maybe it'll be an official spacing in Warp then 😄 Maybe just use a local constant then, just to make sure, this magic number is defined somewhere.

Reply:
yeah, I think it will be 🤞 Will add a local one, thanks 👍